pom.xml

```diff
@@ -142,7 +142,8 @@
     <dropwizard.version>3.1.0</dropwizard.version>
     <dropwizard-metrics-hadoop-metrics2-reporter.version>0.1.2</dropwizard-metrics-hadoop-metrics2-reporter.version>
     <druid.version>27.0.1</druid.version>
-    <flatbuffers.version>1.6.0.1</flatbuffers.version>
+    <esri.version>2.2.4</esri.version>
+    <flatbuffers.version>1.9.0</flatbuffers.version>
```

Comment:
Our arrow version is already set to 2.0.0 therefore we don't need to revert arrow upgrade PR, please cherrypick below to update flatbuffers.version from 1.6 to 1.9
https://issues.apache.org/jira/browse/HIVE-26468

Reply:
I have cherry-picked the mentioned commit.